osbs/api.py

```diff
@@ -41,6 +41,11 @@ def catch_exceptions(*args, **kwargs):
             # Re-raise OsbsExceptions
             raise
         except Exception as ex:
+            # Propogate flexmock errors immediately (used in test cases)
+            if (hasattr(ex, '__module__') and
+                    getattr(ex, '__module__') == 'flexmock'):
```

Comment:
If you pass a third param to `getattr`, it's used as the default value instead of raising an exception. Might make a little bit cleaner.